### PR TITLE
Some technical improvements

### PR DESCRIPTION
**Pull request by hwoarangmy**

### CODE REVIEW

**Comment by akien-mga** [comment]
Good to merge as far as I'm concerned. It runs fine and the code looks ok (though I'll admit I did not review df085d7 thoroughly :p)

File: source/gamemap/GameMap.cpp [hunk 1]
```diff
@@ -619,6 +619,7 @@ Creature* GameMap::getWorkerToPickupBySeat(Seat* seat)
         bool isClaiming = false;
         bool isDigging = false;
 
+#if 0
```
Comment (hwoarangmy) [inline source/gamemap/GameMap.cpp]: I forgot to remove this. Note that this prevents merging.

File: source/gamemap/GameMap.cpp [hunk 2]
```diff
@@ -736,6 +738,7 @@ Creature* GameMap::getFighterToPickupBySeat(Seat* seat)
         bool isFleeing = false;
         bool isBusy = false;
 
+#if 0
```
Comment (hwoarangmy) [inline source/gamemap/GameMap.cpp]: same

**Comment by Bertram25** [comment]
Ok, I'll review all that tonight.

**Comment by hwoarangmy** [comment]
@Bertram25 I'm done with what I wanted to do with the ceature actions ^^
Feel free to review when you can

**Comment by Bertram25** [comment]
Ok :) I'll do that near noon

**Comment by Bertram25** [comment]
I've reviewed the code and it looks good to me. I also had a full game, and everything looks fine so far. (I won but not by far...)
The gameplay has changed a lot, and it's much better. :)
This can be merged IMHO.

**Comment by Bertram25** [comment]
I've merged to move on with this. :)

**Pull request merged**